service/src/test/java/com/bionicuniversity/edu/fashiontips/service/UserServiceTest.java

```diff
@@ -0,0 +1,140 @@
+package com.bionicuniversity.edu.fashiontips.service;
+
+import com.bionicuniversity.edu.fashiontips.entity.User;
+import com.bionicuniversity.edu.fashiontips.service.util.exception.NotFoundException;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlConfig;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static com.bionicuniversity.edu.fashiontips.ServiceTestData.*;
+/**
+ * Created by Sergiy on 11/30/2015.
+ */
+@ActiveProfiles("dev")
+@ContextConfiguration(locations = {
+        "classpath:spring/spring-service.xml",
+        "classpath:spring/spring-persistence.xml"
+})
+@Sql(scripts = {"classpath:db/filloutHSQLDB.sql"},
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        config = @SqlConfig(encoding = "UTF-8"))
+/*Execute tests ordering by method name*/
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
@barik111 This is obviously bad design to make tests dependable on each other. Do you really need to use the userService.getAll method almost in every test?